src/darwin/Framework/CHIPTests/MTRCommissionableBrowserTests.m

```diff
@@ -36,9 +36,27 @@
 // Singleton controller we use.
 static MTRDeviceController * sController = nil;
 
+static NSString * kInstanceNameKey = @"instanceName";
+static NSString * kVendorIDKey = @"vendorID";
+static NSString * kProductIDKey = @"productID";
+static NSString * kDiscriminatorKey = @"discriminator";
+static NSString * kCommissioningModeKey = @"commissioningMode";
+
+static NSDictionary<NSString *, id> * ResultSnapshot(MTRCommissionableBrowserResult * result)
+{
+    return @{
```

Comment:
Could use `dictionaryWithValuesForKeys:...` but this is fine too.

Reply:
I'll stick with this, so I don't have to worry about what dictionaryWithValuesForKeys does with the unboxed boolean...